src/content/deployment/flavors.md

```diff
@@ -114,6 +116,34 @@ build configurations already created.
 </li>
 </ol>
 
+### Manually update podfile
+
+After running `flutter clean`, `flutter pub`, and `pod install`,
+you receive a build error indicating that it is unable to load
+the contents of a file list,
+you will need to manually update your podfile.
+
+To fix this, use the following steps.
+
+1. Rename the podfile schemes as follows:
+
+```yml
+project 'Runner', {
+  'Debug-free' => :debug,
+  'Profile-free' => :release,
+  'Release-free' => :release,
+  'Debug-prod' => :debug,
+  'Profile-prod' => :release,
+  'Release-prod' => :release,
+}
+```
+
+2. Run `flutter clean` and `flutter pub get`.
+
+3. Run `pod install`.
```

Comment:
Flutter developers should never need to run `pod install` (except for add-to-app).  If we have instructions to explicitly run it anywhere else in the docs, we should remove it.
You can delete steps 2 and 3 and go right to "Build the app."